File: src/erbScale/erb.m
%
% Author: Juan M. Fonseca-Solís (juan.fonsecasolis@ucr.ac.cr)
%
function [f] = erb(outImgFolderPath)
    d = 1:36;
    f = (2.^(d./5.7)-1)*230;
    plot(d,f,'linewidth',2)
    xlabel('Distancia desde la cuspide (mm)')
    ylabel('Frecuencia (Hz)')
    xlim([0, 36])
    grid on
    print(strcat(outImgFolderPath, '/escalaERB.pdf'),'-dpdfwrite')

File: src/erbScale/erb.py
# -*- coding: utf-8 -*-
# 2018 Juan M. Fonseca-Solís (juan.fonsecasolis@ucr.ac.cr)
#
from oct2py import octave
import os


def erb(outFolderPath=None):
    octave.addpath(os.path.dirname(os.path.realpath(__file__)))
    octave.erb(outFolderPath)
